vast_csi/builders/block.py

```diff
@@ -321,6 +329,8 @@ def volume_context(self) -> dict:
             context["vip_pool_name"] = self.vip_pool_name
         elif self.vip_pool_fqdn:
             context["vip_pool_fqdn"] = self.vip_pool_fqdn_with_prefix
+        if self.host_encryption:
```

Comment:
Here, logically if self.host_encryption is always true because 
host_encryption can be "true" or "false". Both for python is True strings.

There is helper to convert "true"/"false" to bool 
See: https://github.com/vast-data/vast-csi/blob/v2.6/vast_csi/plugins/csi.py#L330